packages/terra-switch/README.md

```diff
@@ -0,0 +1,45 @@
+# Terra Switch
+
+[![NPM version](https://badgen.net/npm/v/terra-switch)](https://www.npmjs.org/package/terra-switch)
+[![Build Status](https://badgen.net/travis/cerner/terra-switch)](https://travis-ci.com/cerner/terra-core)
+
+The terra-switch component indicates whether a setting is on or off and provide users with instantaneous feedback.
+
+Switches are used to activate settings immediately, in real time.
+Switches never require users to press a button to apply their settings.
+A switch is a set of binary options and always has one of the options selected by default.
+
+Switch should be used for only one of these set of binary options:
+• On / Off
+
+- [Getting Started](#getting-started)
+- [Documentation](https://engineering.cerner.com/terra-core/components/terra-switch/switch/switch)
+- [LICENSE](#license)
+
+## Getting Started
+
+- Install with [npm](https://www.npmjs.com): `npm install terra-switch`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## LICENSE
+
+Copyright 2019 - 2020 Cerner Innovation, Inc.
```

Comment:
This should be only "2020" and not "2019 - 2020"

Reply:
updated :  48fce2d9ca73773ac6f9891feaa7f2a56b36f864